README.md

```diff
@@ -48,7 +48,8 @@ options = {
   columns: 4,
   group: 20,
   gif: false,
-  keep_images: false
+  keep_images: false,
+  basename: 'filename' #this is optional. If not specificed this will be 'sprite'
```

Comment:
I'd prefer this line be changed. So it should instead be:

```
basename: 'sprites', # Base filename for outputs. Optional; default is "sprites".
```

Can you please make this change?

Reply:
@jronallo Done